programs/voter-stake-registry/src/instructions/create_registrar.rs

```diff
@@ -72,26 +72,36 @@ pub fn create_registrar(
     fill_authority: Pubkey,
     distribution_authority: Pubkey,
 ) -> Result<()> {
-    let registrar = &mut ctx.accounts.registrar.load_init()?;
-    require_eq!(registrar_bump, *ctx.bumps.get("registrar").unwrap());
-    registrar.bump = registrar_bump;
-    registrar.governance_program_id = ctx.accounts.governance_program_id.key();
-    registrar.realm = ctx.accounts.realm.key();
-    registrar.realm_governing_token_mint = ctx.accounts.realm_governing_token_mint.key();
-    registrar.realm_authority = ctx.accounts.realm_authority.key();
+    let signers_seeds = {
```

Comment:
please, leave a describing comment here. It looks strange that `signers_seeds` includes all the operations below. I just showed this as an example. Also, you may want to redeclare the block or do sth else.